src/main/java/com/vacation_bot/domain/models/VacationTotalModel.java

```diff
@@ -0,0 +1,50 @@
+package com.vacation_bot.domain.models;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.CompoundIndex;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@Document
```

Comment:
Specify a collection name `vacationTotal` to avoid `Model` ending.

Reply:
Ok.